src/OpenApiLinter.php

```diff
@@ -37,7 +38,11 @@ public function getInfoDescription() {
   }
 
   public function getLinterName() {
-    return 'OPENAPI';
+    return $this->linter_name;
```

Comment:
The linter name is used by [more than just the lint output](https://github.com/phacility/arcanist/blob/f0f95e5b2612d011592f9dbe1885756c5219c70b/src/lint/linter/ArcanistLinter.php#L74-L88), and it should be static.

If you want to modify the name in the output, provide a different value to `setName()` on the message rather than modifying this globally for the linter.